# 0 init 1 allocated 2 explored 3 complete
int32   task_status
int32   task_ID
Point2d task_pos
int32   is_target
